src/neptune/internal/utils/images.py

```diff
@@ -45,6 +45,13 @@
 from neptune.internal.utils.logger import logger
 
 _logger = logging.getLogger(__name__)
+SEABORN_GRID_CLASSES = set(
+    [
+        "FacetGrid",
+        "PairGrid",
+        "JointGrid",
+    ]
+)
```

Comment:
While this is technically correct, the syntax is slightly confusing and IDEs like PyCharm complain about it.
Let's replace it with a set literal:
`SEABORN_GRID_CLASSES = {"FacetGrid", "PairGrid", "JointGrid"}`

Also, what happened to 'pairplot' and 'jointplot'?

Reply:
'pairplot' and 'jointplot' are functions provided to user in [seaborn axisgrid](https://github.com/mwaskom/seaborn/blob/master/seaborn/axisgrid.py) but they always return instances of FacetGrid", "PairGrid", "JointGrid"